maxim/models/dataset.py

```
@@ -100,6 +101,37 @@ def to_json(self):
             }
         return return_dict
 
+    @classmethod
+    def from_json(cls, data: Dict[str, Any]) -> "DatasetEntry":
```

Ruff (0.12.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)